### PR TITLE
(pricing): Update increase in price for 3.5 Haiku.

### PR DESCRIPTION
**Pull request by Manouchehri**
Really confused my previous commit 96362e6213b9a9936fe6f9b26e9a78ef4bb78128 disappeared..

Anyway, here's the increased prices that were announced today for Claude 3.5 Haiku.

### CODE REVIEW

**Comment by Manouchehri** [comment]
I am not sure why this wasn't merged in instead of #6588.

File: litellm/model_prices_and_context_window_backup.json [hunk 1]
```diff
@@ -1730,6 +1730,22 @@
...
+        "max_input_tokens": 200000,
+        "max_output_tokens": 8192,
+        "input_cost_per_token": 0.000001,
+        "output_cost_per_token": 0.000005,
+        "cache_creation_input_token_cost": 0.00000125,
+        "cache_read_input_token_cost": 0.0000001,
+        "litellm_provider": "anthropic",
+        "mode": "chat",
+        "supports_function_calling": true,
+        "supports_vision": true,
```
Comment (tychota) [inline litellm/model_prices_and_context_window_backup.json:1744]: I think this is false.

image

and modifying the map locally, I get error from anthopic API

image

Comment (Manouchehri) [reply litellm/model_prices_and_context_window_backup.json:1744]: Ah you're right.

File: litellm/model_prices_and_context_window_backup.json [hunk 2]
```diff
@@ -1730,6 +1730,22 @@
...
         "supports_prompt_caching": true
     },
+    "claude-3-5-haiku-20241022": {
+        "max_tokens": 8192,
+        "max_input_tokens": 200000,
+        "max_output_tokens": 8192,
+        "input_cost_per_token": 0.000001,
+        "output_cost_per_token": 0.000005,
+        "cache_creation_input_token_cost": 0.00000125,
+        "cache_read_input_token_cost": 0.0000001,
```
Comment (tychota) [inline litellm/model_prices_and_context_window_backup.json:1740]: Not sure about than. Trusting Anthropic docs, this don't seems accurate.

image

**Comment by tychota** [comment]
Apart from my comments, the max_token looks accurate. 
Maybe you can open a PR just to correct this.

File: litellm/model_prices_and_context_window_backup.json (hunk 2 above)
Comment (Manouchehri) [reply litellm/model_prices_and_context_window_backup.json:1740]: I don't see 3.5 Haiku in that screenshot. That's 3 Haiku pricing.

Comment (tychota) [reply litellm/model_prices_and_context_window_backup.json:1740]: Exactly, 3.5 Haiku don't support caching. At least as docs goes.

Later in the doc:
image

(sorry for my screen shot, my communication could be clearer)

Comment (tychota) [reply litellm/model_prices_and_context_window_backup.json:1740]: Reference to doc: https://docs.anthropic.com/en/docs/build-with-claude/prompt-caching#supported-models

Comment (Manouchehri) [reply litellm/model_prices_and_context_window_backup.json:1740]: image

https://www.anthropic.com/pricing#anthropic-api

I think that other page is just outdated.

Comment (tychota) [reply litellm/model_prices_and_context_window_backup.json:1740]: Ah makes senses then !

Comment (tychota) [reply litellm/model_prices_and_context_window_backup.json:1740]: I did write to Anthropic supports to address this.

**Comment by Manouchehri** [comment]
Closing in favor of #6666.

**Pull request closed without merging**